lib/Parse/ParseIfConfig.cpp

```diff
@@ -972,34 +972,25 @@ Result Parser::parseIfConfigRaw(
   return finish(EndLoc, HadMissingEnd);
 }
 
-/// Parse and populate a #if ... #endif directive.
```

Comment:
Is this intentional?

Reply:
... nope, accident of refactoring. Thank you!

Reply:
Oh, this is the .cpp file. This function is documented in the header, which is good enough. We don't need to duplicate the comment here.